packages/nimble-components/src/rich-text-editor/index.ts

```diff
@@ -347,8 +481,83 @@ export class RichTextEditor extends FoundationElement {
                 return false;
         }
     }
+
+    private unbindEditorUpdateEvent(): void {
+        this.tiptapEditor.off('update');
+    }
+
+    /**
+     * input event is fired when there is a change in the content of the editor.
+     *
+     * https://tiptap.dev/api/events#update
+     */
+    private bindEditorUpdateEvent(): void {
+        this.tiptapEditor.on('update', () => {
+            this.$emit('input');
+            this.queueUpdateScrollbarWidth();
+        });
+    }
+
+    /**
+     * Stopping the native input event propagation emitted by the contenteditable element in the Tiptap
+     * since there is an issue(linked below) in ProseMirror where selecting the text and removing them
+     * does not trigger the native HTMLElement input event. So using the "update" event emitted by the
+     * Tiptap to capture it as an "input" customEvent in the Rich text editor.
+     *
+     * Prose mirror issue: https://discuss.prosemirror.net/t/how-to-handle-select-backspace-delete-cut-type-kind-of-events-handletextinput-or-handledomevents-input-doesnt-help/4844
+     */
```

Comment:
nit: minor copy editing
```suggestion
    /**
     * Stopping the native input event propagation emitted by the contenteditable element in the Tiptap
     * since there is an issue (linked below) in ProseMirror where selecting the text and removing it
     * does not trigger the native HTMLElement input event. So using the "update" event emitted by the
     * Tiptap to capture it as an "input" customEvent in the rich text editor.
     *
     * Prose Mirror issue: https://discuss.prosemirror.net/t/how-to-handle-select-backspace-delete-cut-type-kind-of-events-handletextinput-or-handledomevents-input-doesnt-help/4844
     */
```